lib/core/application/backend.ts

```diff
@@ -571,6 +571,10 @@ export class Backend {
       writable: true
     });
 
+    Reflect.defineProperty(global, 'app', {
```

Comment:
An error should be raised (or even better, an assertion) if the `app` property is already defined